src/monsters.test.c

```diff
@@ -185,11 +188,83 @@ static void test_dragon_attacks_the_rich(CuTest * tc)
     test_cleanup();
 }
 
+typedef struct msg_info {
+    const char *name;
+    int numpar;
+    const char **argv;
+    const char **pnames, **tnames;
+    variant *values;
+} msg_info;
+
+static msg_info *test_create_messagetype(const struct locale *lang, const char *name, int num_args, ...) {
+    va_list vargs;
+    const message_type *mtype;
+    char zBuffer[128];
+    int i;
+    msg_info *info = malloc(sizeof(msg_info));
+    info->argv = calloc(num_args + 1, sizeof(char *));
+    info->pnames = calloc(num_args, sizeof(char *));
+    info->tnames = calloc(num_args, sizeof(char *));
+    info->values = calloc(num_args, sizeof(variant_type));
```

Comment:
auch diese calloc Aufrufe haben kein free. Generell finde ich hier, dass Du eine Menge komplexen Code schreibst, nur um etwas zu testen, das ist in der Regel ein schlechtes Zeichen (und varargs sind auch ein code smell).